org-owners-guide.md

```diff
@@ -47,8 +47,36 @@ Note, this list is subject to change.
 There are more organization names that we are squatting on with possible future
 intentions. [For more details please see community issue #1407](https://github.com/kubernetes/community/issues/1407).
 
+## Team Guidance
+
+Each organization should have the following teams:
+
+- teams for each repo `foo`
+  - `foo-admins`: granted admin access to the `foo` repo
+  - `foo-maintainers`: granted write access to the `foo` repo
+- a `bots` team
+  - should contain bots such as @k8s-ci-robot and @linuxfoundation that are
+    neccessary for org and repo automation
+- an `owners` team
+  - should be populated by everyone who has `owner` privileges to the org
+  - gives users the opportunity to ping owners as a group rather than having to
+    search for individuals
+
+**NB**: Not all organizations in use today currently follow this team guidance.
+We are looking to coalesce existing teams towards this model, and use this model
+for all orgs going forward.  Notable discrepancies at the moment:
+
+- `foo-reviewers` teams that grant read access to each `foo` repo haven't
```

Comment:
discussed during steering committee meeting, and we decided this pattern would be useful for the kubernetes-sigs org, so... nevermind, I'll take this back

assuming in-repo or multi-repo subprojects work out, it'd probably make more sense to have `subproject-reviewers` teams that correspond to subprojects rather than repos